lib/utils.py

```diff
@@ -83,7 +84,8 @@ def get_node_ip():
         requests.ConnectionError,
         requests.ConnectTimeout,
         requests.Timeout,
-    ), tries=3)
```

Comment:
Did you intentionally lose the retry count?

Reply:
Yes, I did it on purpose so that it'll keep retrying...

Reply:
That's fair.